### PR TITLE
remove cert manager

### PR DESCRIPTION
**Pull request by haoqing0110**
Replace management-ingress service certificate created by cert-manager 
https://github.com/open-cluster-management/backlog/issues/10455
https://github.com/open-cluster-management/backlog/issues/10826

### CODE REVIEW

**Comment by haoqing0110** [comment]
Fresh install testing result:
Screen Shot 2021-03-24 at 4 34 25 PM

File: stable/management-ingress/values.yaml [hunk 1]
```diff
@@ -29,14 +29,6 @@ tolerations:
   key: node-role.kubernetes.io/infra 
   operator: Exists
 
-cert:
```
Comment (clyang82) [inline stable/management-ingress/values.yaml:32]: Have you checked if it is using by `multicloudhub-operator` or not?

Comment (haoqing0110) [reply stable/management-ingress/values.yaml:32]: @clyang82 these values are used as a reference in certificate CR, and will still be created, do you mean we also need to remove below resources?
```
#  oc get certificates.certmanager.k8s.io multicloud-ca-cert
NAME                 READY   SECRET               AGE   EXPIRATION
multicloud-ca-cert   True    multicloud-ca-cert   21h   2026-03-23T08:00:58Z
# oc get issuers.certmanager.k8s.io multicloud-ca-issuer
NAME                   AGE
multicloud-ca-issuer   21h
```

Comment (clyang82) [reply stable/management-ingress/values.yaml:32]: That's fine. it should be removed along with cert-manager.

**Pull request merged**